### PR TITLE
FHIR Stack Local for e2e Testing

### PR DESCRIPTION
**Pull request by jonahkaye**
Refs: #[1442](https://github.com/metriport/metriport-internal/issues/1442)

### Description

- an additional fhir stack local for testing
- related to https://github.com/metriport/metriport/pull/1583

### Release Plan

- [ ] merge this

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,6 +24,11 @@ You can use the `docker-compose` to run the app and DB servers by executing the
 $ docker-compose up --build
 ```
 
+To run the secondary app and Servers for testing run:
+```shell 
+$ docker-compose -p datamapper -f Docker-compose-data-mapper.yml up -d
```
Comment (leite08) [inline README.md:29]: nit: lower case on the file

nit: while `docker-compose` works, the recommended way to do it now is to use `docker compose ...` (compose is part of docker cmd) - [reference](https://docs.docker.com/compose/migrate/)

File: docker-compose-data-mapper.yml [hunk 2]
```diff
@@ -0,0 +1,35 @@
+version: "3"
+services:
+  fhir-server-data-mapper:
```
Comment (leite08) [inline docker-compose-data-mapper.yml:3]: ditto: data-mapper > fhir-converter

**Review by leite08 — COMMENTED** [review]
Thanks for doing this!

The main thing I'd suggest is to keep consistency with names: this is being used by the FHIR Converter, so I think we should name it accordingly. "data-mapper" is a generic term and name on the Slack channel that encompasses multiple things, like the FHIR Converter, the Terminology Server, maybe more? :)

Actually, we could name it "e2e" so we can use this new instance for whatever e2e test case/situation we want, not only tied to a specific app... but to the fact this instance is not to contain any data.

nit: add a link to the related PR.

File: README.md [hunk 3]
```diff
@@ -24,6 +24,11 @@ You can use the `docker-compose` to run the app and DB servers by executing the
 $ docker-compose up --build
 ```
 
+To run the secondary app and Servers for testing run:
```
Comment (leite08) [inline README.md:27]: I'd add a bit more context here, indicating why we have this secondary instance and what happens with it, a big WARNING indicating data there is nuked as part of the usage, so it shouldn't be used for any purpose that expect data to persist for more than a few minutes.

**Review by jonahkaye — COMMENTED** [review]
I thought fhir converter made the most sense as the convention to go with here re-naming. 
According to the definition we have been going with, the testing we have been doing with the fhir-converter have been Integration tests not unit tests I think. But anyway since its still really just a helpful thing for local stack, I would say its not really an integration test either 😆

**Review by leite08 — APPROVED** [review]
:shipit:

**Pull request merged**